theme/ckeditor5-ui/components/list/list.css

```diff
@@ -79,3 +84,81 @@
 	width: 100%;
 	background: var(--ck-color-base-border);
 }
+
+.ck .todo-list {
+	list-style: none;
+}
+
+.ck .todo-list > li {
+	margin-bottom: 5px;
+}
+
+.ck .todo-list > li > .todo-list {
+	margin-top: 5px;
+}
+
+/*
+ * Let's hide native checkbox and make a fancy one.
+ *
+ * Note: Checkbox element cannot be hidden using `display: none` or `visibility: hidden`.
+ * It has to be clickable to not steal the focus after clicking on it.
+ */
+.ck .todo-list__checkmark input[type='checkbox'] {
+	display: block;
+	width: 100%;
+	height: 100%;
+	opacity: 0;
+	margin: 0;
+}
+
+.ck .todo-list__checkmark {
+	cursor: pointer;
+	width: var(--ck-todo-list-checkmark-size);
+	height: var(--ck-todo-list-checkmark-size);
+	position: relative;
+	display: inline-block;
+	vertical-align: middle;
+	left: -23px;
+	margin-right: -15px;
```

Comment:
Can we use one of `--ck-spacing-` instead (with a similar value)? This value does not work well when text is huge; it's too close to the checkmark

![image](https://user-images.githubusercontent.com/1099479/65053741-b782aa80-d96c-11e9-9d4e-366228606a23.png)

Reply:
The biggest spacing (--large) is still too small, I've decided to use:
```css
left: calc( var(--ck-spacing-large) * -1.5 );
```
WDYT?